.changeset/rude-bees-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@directus/app": patch
+---
+
+Allow batchMode and batchActive to be used in interface extensions
```

Comment:
We use past tense for changsets 👍 
```suggestion
Allowed batchMode and batchActive to be used in interface extensions
```

Reply:
I'll fix that on main so we don't have to delay the merge/launch 👍🏻

Reply:
@paescuj On second thought, I think this is a case where present tense makes more sense! "Allowed to do xyz" sounds like you could do it, but not anymore, whereas "Allow xyz" feels like "We started allowing you do to xyz" 🤔 

Bug fixes are different though, as "Fix issue 123" sounds like something you're about to do, whereas "Fixed 123" sounds like it's done and included 🙂

Reply:
Haha yeah agreed 100% 😄